### PR TITLE
Convert container to integer if it is not None.

### PR DESCRIPTION
**Pull request by truongvan**

### CODE REVIEW

**Review by codingjoe — APPROVED** [review]
LGTM, thanks @truongvan

File: pictures/contrib/rest_framework.py [hunk 1]
```diff
@@ -34,12 +35,14 @@ def to_representation(self, obj: PictureFieldFile):
...
         try:
-            query_params = self.context["request"].GET
+            query_params: QueryDict = self.context["request"].GET
         except KeyError:
             pass
         else:
             ratio = query_params.get(f"{self.source}_ratio")
             container = query_params.get(f"{self.source}_container")
+            if container is not None:
+                container = int(container)
```
Comment (codingjoe) [inline pictures/contrib/rest_framework.py:45]: On second thought, would you mind adding a test for this line?

Comment (truongvan) [reply pictures/contrib/rest_framework.py:45]: I added the test here #95

**Comment by codingjoe** [comment]
Hi @truongvan, same thing here. I took the liberty to extend your patch with a couple more tests and more extensive error handling. I again credited you as a co-author. I will release both patches now. Thanks again for your contribution. I really appreciate it. However, please be so kind and open a ticket next time before submitting a pull-request. It makes it much easier for me to review and asses patches. Best Joe

**Pull request merged**